orchestrator/careplanservice/service.go

```diff
@@ -695,6 +695,16 @@ func (s *Service) validateLiteralReferences(ctx context.Context, resource any) e
 		return err
 	}
 
+	// Make a list of allowed FHIR base URLs, normalize them to all make them end with a slash
+	fhirBaseURLs, err := s.profile.CsdDirectory().LookupEndpoint(ctx, nil, profile.FHIRBaseURLEndpointName)
```

Comment:
If we have great success, it could become a problem; for now there's will be max. a handful of care organizations (and thus entries) any ways. Let's see what needs to be optimized if this becomes a bottleneck.